### PR TITLE
Use iamResource name for condition keys

### PR DESCRIPTION
**Pull request by kstich**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/source/1.0/spec/aws/aws-core.rst [hunk 1]
```diff
@@ -1525,7 +1524,7 @@ HTTP header. Thus, the resolved location is always ``Header``.
 
 .. _Header-based auth: https://docs.aws.amazon.com/AmazonS3/latest/API/sig-v4-header-based-auth.html
 .. _Streaming-signing auth: https://docs.aws.amazon.com/AmazonS3/latest/API/sigv4-streaming.html
-.. _chunked trailer part: https://tools.ietf.org/html/rfc7230#section-4.1.2
+.. _chunked trailer part: https://docs.aws.amazon.com/AmazonS3/latest/API/sigv4-streaming.html
```
Comment (adamthom-amzn) [inline docs/source/1.0/spec/aws/aws-core.rst:1527]: should this be a separate commit (I'm fine without a separate PR)

**Review by adamthom-amzn — APPROVED** [review]

File: docs/source/1.0/spec/aws/aws-core.rst (hunk 1 above)
Comment (kstich) [reply docs/source/1.0/spec/aws/aws-core.rst:1527]: Probably should have been, but not worth a ton of effort to correct either.
